### DISCUSSION TITLE
How to delete Python files in one go with Pybricks Code

### DISCUSSION THREAD

**Discussion by kubotafamily** [pybricks/support#1782] [Q&A] [answered]
The Python files I've been creating with Pybricks Code have piled up.

Is there a way to delete them all at once?

It's a pain to have to click the trash button one by one to delete them.

**Comment by dlech** [comment, marked as answer]
There isn't an easy way to do this. But if you really want do delete ALL scripts, you can delete the database where they are stored using the web developer tools. 

Press <kbd>F12</kbd>, select *Application* tab, select *Storage* > *IndexedDB* > *pybricks.FileStorage* in tree, click *Delete Database* button, Press <kbd>CTRL</kbd>+<kbd>R</kbd> to reload the page

![image](https://github.com/user-attachments/assets/69b6df5f-f015-4304-8d10-d18d97b64c94)

And I would backup the old files first in case you have regrets later about deleting them. :smile:

**Reply by kubotafamily** [reply]
Thank you for your answer!

Thanks to this, I was able to delete all the Python files at once.
